packages/google_maps_flutter/google_maps_flutter_web/README.md

```diff
@@ -46,3 +46,5 @@ There's no `defaultMarkerWithHue` in web. If you need colored pins/markers, you
 Indoor and building layers are still not available on the web. Traffic is.
 
 Only Android supports "[Lite Mode](https://developers.google.com/maps/documentation/android-sdk/lite)", so the `liteModeEnabled` constructor argument can't be set to `true` on web apps.
+
+Google Maps web uses `HtmlElementView` to render maps. When a `GoogleMap` is stacked below other widgets, [`package:pointer_interceptor`](https://www.pub.dev/packages/pointer_interceptor) must be used to capture mouse events on the Flutter overlays. See issue [#73830](https://github.com/flutter/flutter/issues/73830)
```

Comment:
This should probably be "Google Maps for web".

Reply:
Indeed. Fixed!